engine/chainstate/arc_deafult.go

```diff
@@ -0,0 +1,10 @@
+package chainstate
+
+func defaultArcConfig() *broadcastConfig {
+	return &broadcastConfig{
+		protocol: "arc",
+		minerAPIs: []string{
+			"https://example.com",
```

Comment:
Is it correct address as a default api ?

Reply:
Nazarii is right - default API should be configured for Taal's Arc.

Reply:
default API changed to address of Taal's Arc.